datasets/polsum/polsum.py

```diff
@@ -0,0 +1,192 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Polish Summaries Corpus: the corpus of Polish news summaries"""
+
+from __future__ import absolute_import, division, print_function
+
+import glob
+import xml.etree.ElementTree as ET
+
+import datasets
+
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{
+    ogro:kop:14:lrec,
+    author = "Ogrodniczuk, Maciej and Kopeć, Mateusz",
+    pdf = "http://nlp.ipipan.waw.pl/Bib/ogro:kop:14:lrec.pdf",
+    title = "The {P}olish {S}ummaries {C}orpus",
+    pages = "3712--3715",
+    crossref = "lrec:14"
+}
+@proceedings{
+    lrec:14,
+    editor = "Calzolari, Nicoletta and Choukri, Khalid and Declerck, Thierry and Loftsson, Hrafn and Maegaard, Bente and Mariani, Joseph and Moreno, Asuncion and Odijk, Jan and Piperidis, Stelios",
+    isbn = "978-2-9517408-8-4",
+    title = "Proceedings of the Ninth International {C}onference on {L}anguage {R}esources and {E}valuation, {LREC}~2014",
+    url = "http://www.lrec-conf.org/proceedings/lrec2014/index.html",
+    booktitle = "Proceedings of the Ninth International {C}onference on {L}anguage {R}esources and {E}valuation, {LREC}~2014",
+    address = "Reykjavík, Iceland",
+    key = "LREC",
+    year = "2014",
+    organization = "European Language Resources Association (ELRA)"
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+Polish Summaries Corpus: the corpus of Polish news summaries.
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "http://zil.ipipan.waw.pl/PolishSummariesCorpus"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY v.3"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "http://zil.ipipan.waw.pl/PolishSummariesCorpus?action=AttachFile&do=get&target=PSC_1.0.zip"
+
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class Polsum(datasets.GeneratorBasedBuilder):
+    """Polish Summaries Corpus: the corpus of Polish news summaries."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "id": datasets.Value("string"),
+                "date": datasets.Value("string"),
+                "title": datasets.Value("string"),
+                "section": datasets.Value("string"),
+                "authors": datasets.Value("string"),
+                "body": datasets.Value("string"),
+                "summaries": datasets.features.Sequence(
+                    {
+                        "ratio": datasets.Value("string"),
```

Comment:
this must be an integer right ?

You'll need to regenerate the dataset_infos.json file after this change